pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -302,6 +319,47 @@ func (g *GadgetTracerManager) run() {
 	}
 }
 
+func (g *GadgetTracerManager) createMap() error {
+	// Create and pin BPF map
```

Comment:
Could you expand this comment to explain the purpose of the map and how it's used in the GTM?

Reply:
I renamed the function and added documentation in the code.